lnrpc/signrpc/signer_server.go

```diff
@@ -446,13 +446,17 @@ func (s *Server) SignMessage(ctx context.Context,
 		},
 	}
 
-	// The signature is over the sha256 hash of the message.
 	var digest [32]byte
-	copy(digest[:], chainhash.HashB(in.Msg))
+	if in.RawMsg {
+		copy(digest[:], in.Msg)
```

Comment:
It looks like the digest is 32 byte anyway, so I guess we could leave the `SignDigest` stay untouched?

Reply:
You're right, fixed.